diff --git a/src/xmlbuilder/json2xml.js b/src/xmlbuilder/json2xml.js
index 04c7755c02f85651b704f9f82f1348cc0a8ae2df..c5493a46548498edff7f8a3e578816afb3e6e8e3 100644
--- a/src/xmlbuilder/json2xml.js
+++ b/src/xmlbuilder/json2xml.js
@@ -1,7 +1,7 @@
 'use strict';
 //parse Empty Node as self closing node
-import buildFromOrderedJs from './orderedJs2Xml.js';
 import getIgnoreAttributesFn from "../ignoreAttributes.js";
+import buildFromOrderedJs from './orderedJs2Xml.js';
 
 const defaultOptions = {
   attributeNamePrefix: '@_',
@@ -130,7 +130,7 @@ Builder.prototype.j2x = function(jObj, level, ajPath) {
           else val += this.indentate(level) + '<' + key + '/' + this.tagEndChar;
           // val += this.indentate(level) + '<' + key + '/' + this.tagEndChar;
         } else if (typeof item === 'object') {
-          if(this.options.oneListGroup){
+          if(this.options.oneListGroup || key === "ChildItems"){
             const result = this.j2x(item, level + 1, ajPath.concat(key));
             listTagVal += result.val;
             if (this.options.attributesGroupName && item.hasOwnProperty(this.options.attributesGroupName)) {
@@ -140,7 +140,7 @@ Builder.prototype.j2x = function(jObj, level, ajPath) {
             listTagVal += this.processTextOrObjNode(item, key, level, ajPath)
           }
         } else {
-          if (this.options.oneListGroup) {
+          if (this.options.oneListGroup || key === "ChildItems") {
             let textValue = this.options.tagValueProcessor(key, item);
             textValue = this.replaceEntitiesValue(textValue);
             listTagVal += textValue;
@@ -149,7 +149,7 @@ Builder.prototype.j2x = function(jObj, level, ajPath) {
           }
         }
       }
-      if(this.options.oneListGroup){
+      if(this.options.oneListGroup || key === "ChildItems"){
         listTagVal = this.buildObjectNode(listTagVal, key, listTagAttr, level);
       }
       val += listTagVal;
