diff --git a/src/fxb.js b/src/fxb.js
index f437ca8044c1c7369ce2fc72d588d2eac1ffcf0c..77990d43041cbd81ef406a890d0084ce5edd7d6c 100644
--- a/src/fxb.js
+++ b/src/fxb.js
@@ -1,7 +1,7 @@
 'use strict';
 //parse Empty Node as self closing node
-import buildFromOrderedJs from './orderedJs2Xml.js';
 import getIgnoreAttributesFn from "./ignoreAttributes.js";
+import buildFromOrderedJs from './orderedJs2Xml.js';
 
 const defaultOptions = {
   attributeNamePrefix: '@_',
@@ -130,7 +130,7 @@ Builder.prototype.j2x = function (jObj, level, ajPath) {
           else val += this.indentate(level) + '<' + key + '/' + this.tagEndChar;
           // val += this.indentate(level) + '<' + key + '/' + this.tagEndChar;
         } else if (typeof item === 'object') {
-          if (this.options.oneListGroup) {
+          if (this.options.oneListGroup || key === "ChildItems") {
             const result = this.j2x(item, level + 1, ajPath.concat(key));
             listTagVal += result.val;
             if (this.options.attributesGroupName && item.hasOwnProperty(this.options.attributesGroupName)) {
@@ -140,7 +140,7 @@ Builder.prototype.j2x = function (jObj, level, ajPath) {
             listTagVal += this.processTextOrObjNode(item, key, level, ajPath)
           }
         } else {
-          if (this.options.oneListGroup) {
+          if (this.options.oneListGroup || key === "ChildItems") {
             let textValue = this.options.tagValueProcessor(key, item);
             textValue = this.replaceEntitiesValue(textValue);
             listTagVal += textValue;
@@ -149,7 +149,7 @@ Builder.prototype.j2x = function (jObj, level, ajPath) {
           }
         }
       }
-      if (this.options.oneListGroup) {
+      if (this.options.oneListGroup || key === "ChildItems") {
         listTagVal = this.buildObjectNode(listTagVal, key, listTagAttr, level);
       }
       val += listTagVal;
